### DISCUSSION TITLE
Vuetify 3 lib bundle size

### DISCUSSION THREAD

**Discussion by AlexanderSlesarenko** [vuetifyjs/vuetify#17843] [Q&A]
Hello everyone,
I moved my project from Vue2 to Vue3 and as you can see from screenshots Vuetify 3 lib bundle size increased.

Vue 3:
<img width="815" alt="image" src="https://github.com/vuetifyjs/vuetify/assets/10127687/0b1ccea5-520b-4db6-87c6-0673056fe33d">

Vue 2:
<img width="713" alt="image" src="https://github.com/vuetifyjs/vuetify/assets/10127687/a658a7c0-4c88-4d5b-9358-798d13140708">

I import explicitly only those things that I need, no vuetify loader plugin, for example:
`import { createVuetify } from 'vuetify/framework'`
`import { VTextField } from 'vuetify/components'`
`import { aliases, mdi } from 'vuetify/iconsets/mdi-svg'`

Have such aliases:
![telegram-cloud-photo-size-2-5316943318451605012-y](https://github.com/vuetifyjs/vuetify/assets/10127687/9ba00278-58c2-49fe-a16e-4132e2d35c47)

Is it correct or may be I do something wrong?
If you need any additional details I will provide them.

**Comment by NotTsunami** [comment]
> I import explicitly only those things that I need, no vuetify loader plugin, for example:
> import { VTextField } from 'vuetify/components'

I've seen it mentioned a few times in [discussions](https://github.com/vuetifyjs/vuetify/discussions/16958#discussioncomment-5377064) that you shouldn't be importing from `vuetify/components` but rather more explicitly like `vuetify/components/VTextField` for your case instead to reduce unneeded CSS loading (unsure of how impactful that might be as I still haven't made the switch to Vuetify 3 yet as I am waiting for `VDataTable` to be promoted to stable but I hope this helps!)
